changelog/unreleased/SOLR-17986-remove-check-for-CHANGES-txt-edits.yml

```diff
@@ -0,0 +1,8 @@
+# See https://github.com/apache/solr/blob/main/dev-docs/changelog.adoc
+title: Remove check for CHANGES txt edits
```

Comment:
Please remove this yaml file. Edits to ci/cd is not worthy of a separate changelog entry as it does not affect the Solr application.